.golangci.yml

```diff
@@ -13,9 +13,7 @@ linters:
     - vet
     - gofmt
     - goimports
-    - deadcode
-    - varcheck
-    - structcheck
```

Comment:
can you confirm why removing varcheck, structcheck and deadcode? Is this all covered by `unused` I'm assuming?

Reply:
They are deprecated and have moved into unused.